src/site/content/en/blog/mediastreamtrack-insertable-media-processing/index.md

```diff
@@ -0,0 +1,252 @@
+called local, and has as its source input one of the user's cameras or microphones. A non-local
+`MediaStream` may be representing to a media element, like `<video>` or `<audio>`, a stream
+originating over the network, and obtained via the WebRTC
+[`RTCPeerConnection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection) API, or a
+stream created using the Web Audio API
+[`MediaStreamAudioSourceNode`](https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamAudioSourceNode).
+
+A `MediaStream` consists of zero or more
+[`MediaStreamTrack`](https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack) objects,
+representing various audio or video tracks. Each `MediaStreamTrack` may have one or more channels.
+The channel represents the smallest unit of a media stream, such as an audio signal associated with
+a given speaker, like left or right in a stereo audio track.
+
+## What is insertable streams for `MediaStreamTrack`?
+
+The core idea for insertable streams for `MediaStreamTrack` is to expose the content of a
+`MediaStreamTrack` as a collection of [streams](/streams/) (as defined by the WHATWG
+[Streams API](https://streams.spec.whatwg.org/)), which can be manipulated to introduce new
+components.
+
+## Use cases
+
+Use cases for insertable streams for `MediaStreamTrack` include, but are not limited to:
+
+- Video conferencing gadgets like "funny hats" or virtual backgrounds.
+- Voice processing like software [vocoders](https://en.wikipedia.org/wiki/Vocoder).
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | [In Progress][spec]      |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. Origin trial                          | Not started              |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use insertable streams for `MediaStreamTrack`
+
+### Enabling via chrome://flags
+
+To experiment with insertable streams for `MediaStreamTrack` locally, without an origin trial token,
+enable the `#enable-experimental-web-platform-features` flag in `chrome://flags`.
+
+### Feature detection
+
+You can feature-detect insertable streams for `MediaStreamTrack` support as follows.
+
+```js
+if ('MediaStreamTrackProcessor' in window && 'MediaStreamTrackGenerator' in window) {
+  // Insertable streams for `MediaStreamTrack` is supported.
+}
+```
+
+### Core concepts
+
+Insertable streams for `MediaStreamTrack` objects conceptually splits the `MediaStreamTrack` into
+two components:
+
+- The `TrackProcessor`, which consumes a `MediaStreamTrack` object's source and generates a stream
```

Comment:
I can't find `TrackProcessor`. I did find `MediaStreamTrackProcessor`. Is this misnamed?